wbooks/src/app/screens/Home/index.tsx

```diff
@@ -50,7 +54,8 @@ function HomeScreen() {
 
   return (
     <View style={styles.container}>
-      <FlatList keyExtractor={keyExtractor} data={books} renderItem={renderItem} />
+      {book.booksLoading && <ActivityIndicator size="large" color={colorsApp.primary} />}
```

Comment:
Nice. If you have ts errors after the change in the interface, `loading ? activityindicator : flatlist`